### DISCUSSION TITLE
Where's 4.0.0-rc.5 for react-native-elements

### DISCUSSION THREAD

**Discussion by trajano** [react-native-elements/react-native-elements#3588] [Q&A]
In the releases it shows v4.0.0-rc.5 [Latest](https://github.com/react-native-elements/react-native-elements/releases/latest) but it does not appear to be in npmjs which still shows rc.2 which has the peer dependency error that prevents upgrading to Expo 45

Just the @rneui seems to be published.

**Comment by tannerhallman** [comment]
Haven't tried myself, but the npm repository has changed.

Have you tried:

```
"@rneui/base": "^4.0.0-rc.5"
"@rneui/themed": "^4.0.0-rc.5"
```
